help/destinations/catalog/crm/salesforce.md

```diff
@@ -33,17 +33,17 @@ Before activating data to the Salesforce CRM destination, you must have a [schem
 
 Note the following prerequisites in [!DNL Salesforce CRM], in order to export data from Platform to your Salesforce account:
 
-#### You need to have a Salesforce account {#prerequisites-account}
+#### You need to have a [!DNL Salesforce] account {#prerequisites-account}
 
-Go to the Salesforce [trial](https://www.salesforce.com/in/form/signup/freetrial-sales/) page to register and create a Salesforce account, if you do not have one already.
+Go to the [!DNL Salesforce] [trial](https://www.salesforce.com/in/form/signup/freetrial-sales/) page to register and create a [!DNL Salesforce] account, if you do not have one already.
 
-#### Configure a connected app {#prerequisites-connected-app}
+#### Configure a connected app within [!DNL Salesforce] {#prerequisites-connected-app}
 
-Next, you need to configure a [connected app](https://help.salesforce.com/s/articleView?id=sf.connected_app_create.htm&language=en_US&r=https%3A%2F%2Fhelp.salesforce.com%2F&type=5) within your Salesforce account, if you do not have one already. 
+First, you need to configure a [!DNL Salesforce connected app](https://help.salesforce.com/s/articleView?id=sf.connected_app_create.htm&language=en_US&r=https%3A%2F%2Fhelp.salesforce.com%2F&type=5) within your [!DNL Salesforce] account, if you do not have one already. [!DNL Salesforce CRM] will leverage the connected app to connect to [!DNL Salesforce].
```

Comment:
Minor, but there is a missing brackets pair in the hyperlink. This should be: `[[!DNL Salesforce] connected app]`

Reply:
Actually even connected app is salesforce specific terminology. Hence thought it needs to be in the same DNL.